libcloud/compute/drivers/gce.py

```diff
@@ -1733,15 +1733,19 @@ class GCENodeDriver(NodeDriver):
 
     IMAGE_PROJECTS = {
         "centos-cloud": ["centos"],
+        "cos-cloud": ["container-vm"],
```

Comment:
Could we update the image families too? For example, in [the list you linked](https://cloud.google.com/compute/docs/images#os-compute-support), I see that the families associated with cos-cloud are cos-stable, cos-beta and cos-dev.

Reply:
Yeah, I can update the image families.

Reply:
On a side not: would it be possible to generate this list dynamically?

Reply:
It can be interesting to generate the list dynamically if there's a JSON endpoint somewhere. I'm afraid crawling the HTML page is not worth the effort? I don't know.